config/routes.rb

```diff
@@ -547,5 +547,6 @@
   constraints(id: /[^\/]+/) do
     match 'new/hosts/:id' => 'react#index', :via => :get, :as => :host_details_page
   end
+  get 'page-not-found' => 'react#index', :via => :page_not_found
```

Comment:
`:via` is an HTTP method to be used with this, so I guess it is silently ignored here because get is kinda alias for `match .... via: :get`

So please get rid of `:via` :)

Reply:
I guess I meant to  use `:as` instead... :)